service/co/hotwax/common/DecisionRuleServices.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://moqui.org/xsd/service-definition-3.xsd">
+
+    <service verb="clean" noun="RuleGroupRun" authenticate="false" transaction-timeout="600">
+        <description>Deletes RuleGroupRun records older than a specified number of days.</description>
+        <in-parameters><parameter name="daysToKeep" type="Integer" default="60"/></in-parameters>
+        <out-parameters><parameter name="recordsRemoved" type="Long"/></out-parameters>
+        <actions>
+            <script>
+                import org.moqui.context.ExecutionContext
+                import org.moqui.entity.EntityCondition
+                ExecutionContext ec = context.ec
+                Calendar basisCal = ec.user.getCalendarSafe()
+                basisCal.add(Calendar.DAY_OF_YEAR, (int) - daysToKeep)
+                basisTimestamp = new Timestamp(basisCal.getTimeInMillis())
+                recordsRemoved = ec.entity.find("moqui.service.job.ServiceJobRun")
```

Comment:
we should not use the deleteAll, as for bulk data it fails, we should delete the records using iterator

Reply:
Done. Thanks!